admin-panel/src/main/java/net/cryptic_game/backend/admin/endpoints/website_management/faq/FAQEntryEndpoints.java

```diff
@@ -0,0 +1,87 @@
+package net.cryptic_game.backend.admin.endpoints.website_management.faq;
+
+import io.netty.handler.codec.http.HttpResponseStatus;
+import lombok.RequiredArgsConstructor;
+import net.cryptic_game.backend.admin.authentication.AdminPanelAuthenticator;
+import net.cryptic_game.backend.admin.authentication.Permission;
+import net.cryptic_game.backend.admin.data.sql.entities.website_management.faq.FAQEntry;
+import net.cryptic_game.backend.admin.data.sql.repositories.website_management.faq.FAQEntryRepository;
+import net.cryptic_game.backend.base.api.annotations.ApiEndpoint;
+import net.cryptic_game.backend.base.api.annotations.ApiEndpointCollection;
+import net.cryptic_game.backend.base.api.annotations.ApiParameter;
+import net.cryptic_game.backend.base.api.data.ApiResponse;
+import net.cryptic_game.backend.base.api.data.ApiType;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.regex.Pattern;
+
+@RequiredArgsConstructor
+@ApiEndpointCollection(id = "website_management/faq", description = "manages faq", type = ApiType.REST, authenticator = AdminPanelAuthenticator.class)
+public class FAQEntryEndpoints {
+
+    private final FAQEntryRepository faqEntryRepository;
+    private final Pattern xxsFilter = Pattern.compile("<[^- ]|[^- ]>");
```

Comment:
I thin `<[^ ]+>` is better, because there are existing html tags with a `-` in the name

Reply:
make it static and uppercase